Dart/resources/META-INF/plugin.xml

```diff
@@ -153,6 +153,8 @@
     <lang.findUsagesProvider language="Dart" implementationClass="com.jetbrains.lang.dart.ide.findUsages.DartFindUsagesProvider"/>
     <usageTypeProvider implementation="com.jetbrains.lang.dart.ide.findUsages.DartUsageTypeProvider"/>
     <findUsagesHandlerFactory implementation="com.jetbrains.lang.dart.ide.findUsages.DartServerFindUsagesHandlerFactory"/>
+    <fileStructureGroupRuleProvider implementation="com.jetbrains.lang.dart.ide.findUsages.UnitMemberGroupRuleProvider"/>
+    <fileStructureGroupRuleProvider implementation="com.jetbrains.lang.dart.ide.findUsages.ClassMemberGroupRuleProvider"/>
```

Comment:
Yep, all class names related to Smurf support start with Smurf. :) Otherwise it would be difficult to find anything in the repository that contains support for more than 100 technologies.